### ISSUE TITLE
Fuzzy templates when Faster UI selected

### ISSUE THREAD

**Issue by knox-dawson** [mb1986/rm-hacks#284]
I'm at Version 3.9.5.2026, 0.0.10-alpha.5, but I also saw this at Version 3.9.4.2018, 0.0.10-alpha.4.  When I select "Faster UI", my templates are "fuzzy", even after a five-finger refresh.  This appears to affect any template I select, including custom templates.

![fuzzy-template-not-50-percent](https://github.com/mb1986/rm-hacks/assets/154367947/e0de8ad3-884b-46fb-8cad-cf58fa488b5b)![fuzzy-template-50-percent](https://github.com/mb1986/rm-hacks/assets/154367947/f338c5e9-13c1-4b74-8247-ea63df8e72aa)

**Comment by hakunamatatarr** [comment]
Yes, the quality is in general worse, even after full refresh. Reading a detailed PDF becomes really difficult. But I assumed that was just part of the fast UI. That's why I thought it would be nice to have the fast UI in the document toggle, so you could turn it on and off when you need it.

**Comment by atngames** [comment]
I think the faster ui works by changing the screen display mode. We will not have a way to deal with that.

I leave this open as a reminder to add a toggle for faster ui in the toolbar
